astropy/io/misc/asdf/__init__.py

```diff
@@ -25,6 +25,99 @@
 Documentation on the ASDF Standard can be found `here
 <https://asdf-standard.readthedocs.io>`__. Documentation on the ASDF Python
 module can be found `here <https://asdf.readthedocs.io>`__.
+
+Using ASDF With Table I/O
+=========================
+
+ASDF provides readers and writers for `~astropy.table.Table` using the
+:ref:`table_io`. This makes it easy to read and write ASDF files with
+`~astropy.table.Table` data.
+
+Basic Usage
+-----------
+
+Given a table, it is straightforward to write it out to an ASDF file::
+
+    from astropy.table import Table
+
+    # Create a simple table
+    t = Table(dtype=[('a', 'f4'), ('b', 'i4'), ('c', 'S2')])
+    # Write the table to an ASDF file
+    t.write('table.asdf')
+
+The I/O registry automatically selects the appropriate writer function to use
+based on the ``.asdf`` extension of the output file.
+
+Reading a file generated in this way is equally simple using
+`~astropy.table.Table.read`::
+
+    t2 = Table.read('table.asdf')
+
+The I/O registry automatically selects the appropriate reader function based on
+the extension of the input file.
+
+In the case of both reading and writing, if the file extension is not ``.asdf``
+it is possible to explicitly specify the reader/writer function to be used::
+
+    t3 = Table.read('table.zxcv', format='asdf')
```

Comment:
Does doctest pick this up? If so, need to skip.

Reply:
It should not be picked up by doctest.